bears/vcs/git/GitCommitBear.py

```diff
@@ -58,22 +70,44 @@ def run(self,
                 yield Result(self, "HEAD commit has no message.")
             return
 
-        yield from self.check_shortlog(shortlog_length, stdout[0])
+        arguments = locals()
```

Comment:
oops that was some remains to shorten stuff, but I deleted it again (except for this statement)